ArchiSteamFarm/Bot.cs

```diff
@@ -4114,6 +4120,98 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+		private async Task<string> ResponsePrivacy(ulong steamID, string privacySettingsText) {
+			if ((steamID == 0) || string.IsNullOrEmpty(privacySettingsText)) {
+				ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(privacySettingsText));
+				return null;
+			}
+
+			if (!IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			if (privacySettingsText.Length != 6) { // There are only 6 privacy settings
+				return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(privacySettingsText)));
+			}
+
+			foreach (char digit in privacySettingsText) {
+				if ((digit < '1') || (digit > '3')) { // They can be set from 1 to 3 
+					return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(privacySettingsText)));
+				}
+			}
+
+			// Converting digits to enum
+			Steam.PrivacyResponse privacySettings = new Steam.PrivacyResponse {
+				Comments = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[5].ToString()),
+				Settings = new Steam.PrivacyResponse.PrivacySettings {
+					Inventory = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[3].ToString()),
+					InventoryGifts = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[4].ToString()),
+					OwnedGames = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[1].ToString()),
+					Playtime = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[2].ToString()),
+					Profile = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[0].ToString())
+				}
+			};
+
+			// These can't be set to friends only, only public/private
```

Comment:
Once above is done, this should be right in that `foreach` loop, in a `switch` that will handle all invalid settings at-once.